make/test.mk

```diff
@@ -76,6 +76,22 @@ e2e: $(BINDIR)/scratch/kind-exists | $(NEEDS_KUBECTL) $(NEEDS_GINKGO)
 e2e-ci: e2e-setup-kind e2e-setup
 	make/e2e-ci.sh
 
+$(BINDIR)/test/e2e.test: FORCE | $(NEEDS_GINKGO) $(BINDIR)/test
+	$(GINKGO) build --tags e2e_test test/e2e
+	mv test/e2e/e2e.test $(BINDIR)/test/e2e.test
+
+.PHONY: e2e-build
+## Build an end-to-end test binary
+##
+## The resulting binary can be used to execute the end-to-end tests on a
+## computer without the test source files and without Go or Ginkgo installed.
+## For example, the binary can be copied to an OpenShift CRC virtual machine and
+## used to run end-to-end tests against cert-manager that has been installed
+## using OperatorHub.
+##
+## @category Development
+e2e-build: $(BINDIR)/test/e2e.test
```

Comment:
There was already a target for `$(BINDIR)/test`, but nothing seems to use it, so I used it for this.

Equivalent to `bazel build //test/e2e:e2e` which I used to use in the old-days:
 * https://github.com/cert-manager/cert-manager-olm/#run-e2e-tests-on-crc-cluster